src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.test.tsx

```diff
@@ -637,7 +637,6 @@ describe('SavedObjectsTable', () => {
         { force: true }
       );
       expect(component.state('selectedSavedObjects').length).toBe(0);
-      expect(notifications.toasts.addDanger).not.toHaveBeenCalled();
```

Comment:
any reason to remove this assertion?

Reply:
Thanks for spotting the error, I think it is an accidental delete. always reverted.